src/main/java/com/timgroup/statsd/NonBlockingStatsDClient.java

```diff
@@ -53,9 +54,29 @@
  */
 public class NonBlockingStatsDClient implements StatsDClient {
 
-    public static final String DD_DOGSTATSD_PORT_ENV_VAR = "DD_DOGSTATSD_PORT";
-    public static final String DD_AGENT_HOST_ENV_VAR = "DD_AGENT_HOST";
-    public static final String DD_ENTITY_ID_ENV_VAR = "DD_ENTITY_ID";
+    static final String DD_DOGSTATSD_PORT_ENV_VAR = "DD_DOGSTATSD_PORT";
+    static final String DD_AGENT_HOST_ENV_VAR = "DD_AGENT_HOST";
+    static final String DD_ENTITY_ID_ENV_VAR = "DD_ENTITY_ID";
+    private static final String ENTITY_ID_TAG_NAME = "dd.internal.entity_id" ;
+
+    enum Literal {
+        service,
+        env,
+        version,
+        tags;
+        private static final String PREFIX = "dd";
```

Comment:
Just wondering if this prefix is necessary. We mostly have `service:foo` for instance, not sure if it makes sense to introduce a different tag `dd.service` (for instance) for this. I'll let you discuss with your team. Obviously nothing's wrong with the implementation.  Perhaps you did it because of the `dd.internal.entitiy_id` tag we were using which is prepended with `dd`?

Reply:
it's only for only these 4 tags. For other tags it might be different